sentry/src/main/java/io/sentry/ISpan.java

```diff
@@ -169,4 +169,28 @@ ISpan startChild(
    */
   @Nullable
   Object getData(@NotNull String key);
+
+  /**
+   * Set a measurement without unit.
```

Comment:
`l`: I think it would make sense to explain what happens, when you don't provide the unit, and what benefit you have when you provide the unit.